lib/cass_schema/datastore.rb

```diff
@@ -61,10 +61,18 @@ def general_client
 
     # Internal method used by Runner to pass state into the datastore
     def _setup(options = {})
-      @schema_base_path = options[:schema_base_path]
+      self[:schema_base_path] ||= options[:schema_base_path]
```

Comment:
It could be if I create the accessor. I thought Struct adds the accessors for me, but it doesn't seem to be working that way